search-service/src/main/java/de/cxp/ocs/elasticsearch/query/analyzer/QuerqyQueryExpander.java

```diff
@@ -150,8 +164,9 @@ else if (expandedQuery.getUserQuery() != null) {
 	static class TermFetcher extends AbstractNodeVisitor<Node> {
 
 		@Getter
-		List<WeightedWord>	words	= new ArrayList<>();
-		private Occur		occur;
+		//TODO: Add new impl. of QueryStringTerm as e.g. FilterWord and use that with a field in visit(RawQuery)
```

Comment:
TODO comment can be removed :heavy_check_mark:

Reply:
done